app/uk/gov/hmrc/trackingconsentfrontend/config/AppConfig.scala

```diff
@@ -33,8 +33,14 @@ class AppConfig @Inject() (config: Configuration, servicesConfig: ServicesConfig
   private val platformHost: Option[String] =
     config.getOptional[String]("platform.frontend.host")
 
-  private val helpFrontendHost: String =
-    platformHost.map(_ => "").orElse(config.getOptional[String]("help-frontend.host")).getOrElse("")
+  private val localHelpFrontendBaseUrl =
+    config.getOptional[String]("help-frontend.base-url-local-testing")
+
+  private val helpFrontendHost: String = platformHost match {
+    // if on a environment with Platform host set, use relative URLs
+    case Some(host) => ""
+    case _          => localHelpFrontendBaseUrl.getOrElse("")
+  }
```

Comment:
and saying it out loud for myself mostly, having the config in local.conf isn't going to impact any things testing with service manager, because no-one should be spinning up help frontend with service manager (did a quick search and can't see anyone doing that)

Reply:
might be worth documenting why we do it - we think - because of api platform